packages/addons/remote-functions/index.ts

```diff
@@ -0,0 +1,210 @@
+import {
+	colors,
+	defineAddon,
+	defineAddonOptions,
+	isVersionUnsupportedBelow
+} from '@sveltejs/cli-core';
+import { common, exports, object, type AstTypes } from '@sveltejs/cli-core/js';
+import { parseScript } from '@sveltejs/cli-core/parsers';
+import { addToDemoPage } from '../common.ts';
+import { capitalCase, kebabCase, pascalCase } from 'change-case';
```

Comment:
i would use [scule](https://github.com/unjs/scule) for this - its slightly smaller and has stronger types/inference (though we may not make use of that here)

Reply:
I don't have much preference, I'll follow your advice. 
Thank you for looking 👍